packages/core/src/meshDevice.ts

```
@@ -797,7 +808,7 @@ export class MeshDevice {
       clearInterval(this._heartbeatIntervalId);
     }
     this._heartbeatIntervalId = setInterval(() => {
-      this.heartbeat();
+      this.heartbeat().catch(() => {});
```

If we have a catch statement we should be returning or  throwing an error or remove the empty catch statement

---

This is inside an interval, so it was crashing to `UncaughtException`. Not many options we could do there... we could maybe log it, or emit an event.